addons/l10n_in/models/account_journal.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    l10n_in_debit_sequence = fields.Boolean(
```

Comment:
Do we really need this as a setting?  You seemed to say it is always needed.

Reply:
still asking @ajjo-odoo as it would be there by default.

Reply:
> still asking @ajjo-odoo as it would be there by default.

As discussed, it would be great if we could have consistency with credit note.
We may keep check box by default true for both or we can keep false, 
or we may remove and keep true always.